packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -586,11 +586,6 @@ export abstract class BaseService extends Resource
           enable: true,
           rollback: props.circuitBreaker.rollback ?? false,
         } : undefined,
-        alarms: props.deploymentAlarms ? {
```

Comment:
Can we use a `Lazy.Any()` producer here? that will simplify some of our later logic and allow us to just set a 'deploymentAlarms' property on the L2 construct instead of digging down to modify the resource properties directly.